src/Tasks/Microsoft.NET.Build.Tasks/ResourceAssemblyInfo.cs

```diff
@@ -13,5 +16,15 @@ public ResourceAssemblyInfo(string culture, string relativePath)
             Culture = culture;
             RelativePath = relativePath;
         }
+
+        public static ResourceAssemblyInfo CreateFromReferenceSatellitePath(ITaskItem referenceSatellitePath)
+        {
+            string destinationSubDirectory = referenceSatellitePath.GetMetadata("DestinationSubDirectory");
```

Comment:
Yes, it will be broken if we have a satellite item without this metadata.  What should we do in that case?  Throw an exception?  Ignore this satellite item?  Try to use it without knowing its culture?